src/ResultWriter/WaveFieldWriterExecutor.h

```diff
@@ -280,6 +307,8 @@ class WaveFieldWriterExecutor
 
 public:
 	static const unsigned int NUM_LOWVARIABLES = 7;
+	static const unsigned int NUM_INTEGRATED_VARIABLES = 9;
+	static const unsigned int NUM_TOTALLOWVARS = 16;
```

Comment:
`static const unsigned int NUM_TOTALLOWVARS = NUM_LOWVARIABLES + NUM_INTEGRATED_VARIABLES`

In addition, the names `NUM_TOTALLOWVARS` and `NUM_LOWVARIABLES` are a bit misleading. `NUM_LOWVARIABLES` should be the total number of variables and `NUM_PLASTICITY_VARIABLES` should be the name for what was defined as `NUM_LOWVARIABLES` before.